packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -354,49 +360,82 @@ export class Pipeline extends PipelineBase {
     ];
   }
 
-  private ensureReplicationBucketExistsFor(region?: string): s3.IBucket {
-    if (!region) {
-      return this.artifactBucket;
+  private ensureReplicationBucketExistsFor(action: IAction): CrossRegionInfo {
```

Comment:
Rename since this is now returning more than just a bucket

Reply:
Done.